packtivity/asyncbackends.py

```diff
@@ -389,9 +388,7 @@ class CeleryBackend(PythonCallableAsyncBackend):
         def __init__(self, app=None, packconfig_spec=None):
             super(CeleryBackend, self).__init__(packconfig_spec)
             self.app = app or default_celeryapp
-            self.disable_sync = yaml.load(
-                os.environ.get("PACKTIVITY_CELERY_DISABLE_SYNC", "true")
-            )
+            self.disable_sync = os.environ.get("PACKTIVITY_CELERY_DISABLE_SYNC", "true")
```

Comment:
This just removes the `yaml.load`. I'm not sure why that was being called on something that would return booleans, but maybe this changed in the past?